internal/main.go

```diff
@@ -66,6 +83,7 @@ func main() {
 	flag.StringVar(&flags.provider, "provider", "", "The name of the cloud provider")
 	flag.StringVar(&flags.sshKeys, "ssh-keys", "", "Update SSH keys for the given user")
 	flag.BoolVar(&flags.version, "version", false, "Print the version and exit")
+	flag.Var(&flags.overrides, "set", "Set an environment variable, overriding the default/detected variable")
```

Comment:
- Should be in alphabetical order.
- The `attributes` help text calls them `metadata attributes`, so we should be consistent here.
- The help text should make clear that a key=value pair is wanted and that the option can be specified multiple times.